src/Gateway.php

```diff
@@ -267,7 +267,7 @@ public function capture($transaction_id, $amount, $extra = null) {
         $payload = ['amount' => $int_amount];
 
         if (is_array($extra)) {
-            $payload = array_merge_recursive($payload, $extra);
+            $payload['extra'] = $extra;
```

Comment:
can we confirm this is correct for capture ? in official branch they merge extra, https://github.com/fatzebra/PHP-Library/blob/master/src/Gateway.php